src/plugins/stake-plugins/uniswapV2/policies/VelodromeV2StakePolicy.ts

```diff
@@ -259,23 +259,16 @@ export const makeVelodromeV2StakePolicy = (options: UniswapV2LpPolicyOptions): S
 
               // Existing liquidity that may be unstaked due to a previous failed attempt to stake, or some other reason
               const expectedLiquidityAmount = await getExpectedLiquidityAmount(policyInfo, tokenAAllocation)
-
-              // Already have enough LP-tokens to cover needed amount
-              if (gte(lpTokenBalance, expectedLiquidityAmount)) {
-                return { liquidity: expectedLiquidityAmount }
-              }
```

Comment:
I don't get why removing this makes sense.

Reply:
I get it now. Because we're making deposit to the reward pool optional based on if the pool is alive, then this previous logic no longer makes sense.